packages/api-fetch/src/types.ts

```diff
@@ -1,4 +1,5 @@
-export interface ApiFetchRequestProps {
+export interface ApiFetchRequestProps extends RequestInit {
+	// Override headers, we only accept it as an object due to the `nonce` middleware
```

Comment:
Do we need to extend the `RequestInit` interface? If yes, should we remove `method` prop here?

Reply:
Yes we use basically everything from it. I didn't catch that there were other duplicates, I'll clean those up.